# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/qt5x11extras/hotfix-xcb.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- qtx11extras-everywhere-src-5.15.4/src/src.pro.vanilla	2022-09-16 23:36:16.305000000 +0200
+++ qtx11extras-everywhere-src-5.15.4/src/src.pro	2022-09-16 23:36:25.299000000 +0200
@@ -1,5 +1,5 @@
 TEMPLATE = subdirs
 QT_FOR_CONFIG += gui-private
-qtConfig(xcb) {
+#qtConfig(xcb) {
   SUBDIRS += x11extras
-}
+#}
